paddleocr.py

```diff
@@ -539,6 +542,11 @@ def ocr(self, img, det=True, rec=True, cls=True):
         if det and rec:
             ocr_res = []
             for idx, img in enumerate(imgs):
+                img = alpha_to_color(img, alpha_color)
+                if inv:
+                    img = cv2.bitwise_not(img)
+                if bin:
+                    img = binarize_img(img)
```

Comment:
Why is the preprocess only exist in the both det and rec scenario?

Reply:
Because it's can be helpful in both. Are there some other places it'd be useful? 
Please elaborate on your point.

Reply:
1. Could inv/bin be useful in detection-only and rec-only scenarios?
2. How about exposing all the image processing params in the init_args like the postprocessing params? https://github.com/PaddlePaddle/PaddleOCR/blob/fac03876f39bc67acc8eef6d7facb9a2206eeecd/ppstructure/utility.py#L21

Reply:
I understand now. See the next commit for the changes.